user/database-setup.md

```diff
@@ -433,6 +433,22 @@ We advise verifying the validity of the download URL [on ElasticSearch's website
 
 > `sudo` is not available on [Container-based infrastructure](/user/ci-environment/#Virtualization-environments).
 
+### Installing ElasticSearch on trusty container-based infrastructure
+
+ElasticSearch is  not installed by default on the [trusty container-based infrastructure](/user/trusty-ci-environment/)
+but you can install it by adding the following steps to your `.travis.yml`.
+
+```yaml
+env:
+  - ES_VERSION=5.1.1 ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz
+install:
+  - wget ${ES_DOWNLOAD_URL}
+  - tar -xzf elasticsearch-${ES_VERSION}.tar.gz
+  - ./elasticsearch-${ES_VERSION}/bin/elasticsearch &
+script:
+  - wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200
+```
+
```

Comment:
We certainly could. I'm going with if they need something specific they probably know where to find it though.